tests/imgcat

```diff
@@ -74,14 +77,18 @@ while [ $# -gt 0 ]; do
         show_help
         exit
         ;;
+    -p|--p|--print)
+        print_filename=1
+        ;;
     -*)
         error "Unknown option flag: $1"
         show_help
         exit 1
       ;;
     *)
         if [ -r "$1" ] ; then
-            print_image "$1" 1 "$(base64 < "$1")"
+            has_stdin=f
+            print_image "$1" 1 "$(base64 < "$1")" "$print_filename"
```

Comment:
Also, on line 103, pass `""` to `print_image` since it now takes four arguments.